ci_framework/roles/hotfix/README.md

```diff
@@ -0,0 +1,33 @@
+# hotfix role
```

Comment:
Thank you @son-vyas for working on this. By seeing the role content, I think we donot need an extra role for tcib hotfix utility. We can add the tcib hotflix utility with in [build_containers](https://github.com/openstack-k8s-operators/ci-framework/tree/main/ci_framework/roles/build_containers) role.

Reply:
done